src/crush/CrushWrapper.h

```diff
@@ -1339,6 +1339,13 @@ class CrushWrapper {
       auto &carg = cmap.args[bidx];
       carg.ids = NULL;
       carg.ids_size = 0;
+      if (b) {
+        carg.ids_size = b->size;
+        carg.ids = (__s32*)calloc(4, b->size);
+        for (unsigned i = 0; i < b->size; ++i) {
+          carg.ids[i] = b->items[i];
+        }
+      }
```

Comment:
I left this off deliberately since we don't have any users yet.  When we do, we'll want to distinguish between choose_args that do and don't have ids... but for now we can leave them off all the time.